Sources/Subs-Admin.php

```diff
@@ -529,6 +529,18 @@ function updateSettingsFile($config_vars, $keep_quotes = null, $rebuild = false)
 			'default' => 'SMFCookie11',
 			'type' => 'string',
 		),
+		'auth_secret' => array(
+			'text' => implode("\n", array(
+				'/**',
+				' * Secret key used to create and verify cookies, tokens, etc.',
+				' * Do not change this, and NEVER share it.',
+				' *',
```

Comment:
Wait, shouldn't this say something along the lines of changing this will instantly log out all users?  It should be changeable in the event the forum was breached to invalidate all cookies.

Reply:
Yeah, that'd be better. I'll change it.